documentation/development-processes-checklists.md

```diff
@@ -69,12 +75,13 @@ At least three days prior to the end of the sprint, we will create a `release` b
 1. On Heroku, change staging app to point to this new release branch.
 1. Broadcast publishing of testing app and seek feedback and/or bug reports. Any bugs should be reported to the `#bugs` room in slack and will be triaged from there.
 1. Merge any applicable bugfix branches into `release/#.#.#`.
-    - Include summary of `Merge bugfix/### (#[PRNumber])`
+    - Include summary of `Merge bugfix/ (#[PRNumber])`
```

Comment:
I smell a find-and-replace whoopsie!!! 😃

Reply:
`cmd + d` did it's job, but you are correct ☹️